Story/StoryMenu.cs

```diff
@@ -591,6 +595,10 @@ public void SetChecked(CheckBox box, bool c)
                 {
                     resetSave = !resetSave;
                 }
+                if (idstring == "FRIENDLYFIRE")
+                {
+                    gameMode.friendlyFire = !gameMode.friendlyFire;
```

Comment:
this really should be `gameMode.friendlyFire = c;` shouldn't it

Reply:
worked 👍